setup.py

```diff
@@ -33,16 +33,17 @@
     ],
     install_requires=[
         'panoramic-auth>=0.2.0',
-        'requests>=2.24.0',
-        'urllib3>=1.26.1',
+        'requests<=2.24.0',
+        'urllib3<1.26',
```

Comment:
With the update to the requests version, you can remove this line. This was only needed when requests went up to 2.25.0.

Reply:
Removed.